Sources/SafeDICore/Visitors/InstantiableVisitor.swift

```diff
@@ -33,14 +33,14 @@ public final class InstantiableVisitor: SyntaxVisitor {
 
     public override func visit(_ node: VariableDeclSyntax) -> SyntaxVisitorContinueKind {
         guard
-            declarationType.isTypeDefinition
-            && node.modifiers.staticModifier == nil
+            declarationType.isTypeDefinition,
+            node.modifiers.staticModifier == nil
         else {
             return .skipChildren
         }
         // Check attributes and extract dependency source.
         let dependencySources = node.attributes.dependencySources
-        guard dependencySources.isEmpty || dependencySources.count == 1 else {
+        if dependencySources.count > 1 {
```

Comment:
we no longer `guard` because we want to fall through now and generate all the diagnostics below where applicable.